weather

```diff
@@ -0,0 +1,16 @@
+api_key = 'eb3f523c4bedada94d5846340c5dbde7'
```

Comment:
❗ 
Shouldn't be committing the API key directly. Have a separate config file that you import from. The config file should not be committed either since it would contain the key. 

See the routemodel/data_retrieval folder in the repository for an example of how we keep the API key secure.